### PR TITLE
remove epsilon

### PR DESCRIPTION
**Pull request by leijurv**
Replace epsilon with double min value

### CODE REVIEW

File: Util.java [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
 
 /**
  * Common utilities
  */
 public class Util {
-	/**
-	 * A constant for dealing with floating point errors.
-	 * Add/subtract this from floats when doing equality comparisons.
-	 */
-	public static final double EPSILON = 0.0000001;
```
Comment (c0d3rman) [inline Util.java]: Please remove this line

Comment (ajnadel) [inline Util.java]: @c0d3rman Isn't that line removed?

File: Util.java [hunk 2]
```diff
@@ -22,7 +18,7 @@
 	 * 		Whether or not it is within Constants.EPSILON of zero
```
Comment (ajnadel) [inline Util.java]: @leijurv Please adjust this line as well.

Comment (leijurv) [inline Util.java]: k

**Comment by c0d3rman** [comment]
@ajnadel, your requested change has been made

**Pull request merged**